docs/configuration.rst

```diff
@@ -527,9 +527,9 @@ The following environment variables for the tracer are supported:
          Sets the mode for the automated user login events tracking feature which sets some traces on each user login event. The
          supported modes are ``safe`` which will only store the user id or primary key, ``extended`` which will also store
          the username, email and full name and ``disabled``. Note that this feature requires ``DD_APPSEC_ENABLED`` to be 
-         set to ``true`` to work.  
+         set to ``true`` to work. Currently only supported in Django.
```

Comment:
This note would be more resilient to future changes if it was on the `1.17.0` item under `version_added`.
```suggestion
         set to ``true`` to work.
```